app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-form-tracker.js

```diff
@@ -82,8 +83,12 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
         input.answer = labelText
       } else if (inputNodename === 'SELECT' && elem.options[elem.selectedIndex].value) {
         input.answer = elem.options[elem.selectedIndex].text
-      } else if (inputType === 'text' && elem.value) {
-        input.answer = '[REDACTED]'
+      } else if ((inputType === 'text' || inputType === 'search') && elem.value) {
+        if (this.includeTextInputValues) {
+          input.answer = elem.value
```

Comment:
Quick question: I've checked the implementation guide and it mentions that the `text` property should have a "feasible level of PII protection applied". Do we need to pass this through `PIIRemover`? Although I seem to remember someone mentioning that GA4 might also have some PII removal so might that be enough? I've sent a message to the PA's to check.

Reply:
That's a good shout and an easy fix, so I've implemented it 👍